### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by mhk197**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -16,31 +16,17 @@ Jump to:
 
 ## Summary
 
-We will add the capability of inferring with a Mixture of Experts model (a transformer with MoE, for MLP layers only) to the FlexFlow Serve framework. Once we are able to infer with an MoE model in FlexFlow, we will benchmark its per-token latency and throughput on a node with four V100 GPUs against a standard, non-accelerated service setting.
+We will add support for Mixture of Experts (MoE) Transformer models to the FlexFlow Serve framework. Once we are able to successfully run inference with the model on FlexFlow, we will benchmark its per-token latency and throughput on a node with four V100 GPUs against a standard, non-accelerated service setting.
```
Comment (hugolatendresse) [inline index.md]: I would specify it's MoE for MLP layers only. MoE is possible for other layers

File: index.md [hunk 2]
```diff
@@ -55,79 +41,58 @@ Temporal locality is high for the weights, but very limited for the activations.
...
 with a model that cannot fit on a single GPU necessitates expensive communication, increasing the communication-to-computation ratio. 
-There are infinitely many ways to decompose and schedule the calculation on multiple workers. Experimentation is needed to find the strategy that minimizes 
-the amount of communication (and other costs). We will try to experiment with different techniques, including 
-intra-expert parallelism and inter-expert parallelism, and choose which one performs best. 
+There are many possible ways to decompose and schedule the calculation on multiple workers, and experimentation is needed to find the strategy that minimizes communication and latency. One of the key decision points will be determining how to parallelize the execution of experts that are concurrently activated.
 
 Divergent execution is typical to neural networks due to the non-linear activation functions. In MoE models, an additional 
 layer of divergence is introduced due to the routing to experts. That can introduce load imbalance, as some experts may receive more inputs than others. 
 
-TODO Describe constraints: What are the properties of the system that make mapping the
```
Comment (hugolatendresse) [inline index.md:65]: those are active TODOs, please revert them all!

**Review by hugolatendresse — CHANGES_REQUESTED** [review]

File: index.md (hunk 2 above)
Comment (mhk197) [reply index.md:65]: reverted

**Review by hugolatendresse — APPROVED** [review]

File: index.md (hunk 1 above)
Comment (mhk197) [reply index.md]: added sentence afterwards

**Pull request merged**